gen/gen.go

```diff
@@ -4,15 +4,16 @@ import (
 	"bytes"
 	"embed"
 	_ "embed"
-	"fmt"
-	"slices"
 	"text/template"
 )
 
 type (
 	Driver string
 	// Config passed to template parser
 	Config struct {
```

Comment:
I think its better to store to structs in config, 1 for migrate, and 1 for schema:

```go
Config struct {
  Migrate *migrateConfig
  Schema *schemaConfig
  // global vars
  Env          string
  ...
}
```

Reply:
@ronenlu i thought about it, but after trying, I didn’t see it bringing a huge advantage compared to using a single struct, and it increased code complexity.